impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -15,6 +15,43 @@
 
 namespace impeller {
 
+// Maximum size to use VMA image suballocation. Any allocation greater than or
+// equal to this value will use a dedicated VkDeviceMemory.
+constexpr size_t kImageSizeThresholdForDedicatedMemoryAllocation =
```

Comment:
As mentioned in https://github.com/flutter/engine/pull/43313#discussion_r1246168700, let's put the tunables in a separate file so everything is available at a glance?